src/engine/SCons/Tool/msvsTests.py

```diff
@@ -32,6 +32,7 @@
 import TestCmd
 import TestUnit
 
+from SCons.Script import Dir
```

Comment:
Likely should be SCons.Node.FS import Dir..

Reply:
I haven't been able to get this one to work. I was able to use env.Dir() in msvs.py but not here. I want to use a Dir class without creating the directory and without the directory existing and only SCons.Script.Dir seems to fulfill that; SCons.Node.FS.Dir either creates the directory or throws an exception when it doesn't exist.